slack.json

```diff
@@ -4931,7 +4931,7 @@
             },
             {
                 "module": "slack_sdk",
-                "input_file": "wheels/shared/slack_sdk-3.31.0-py2.py3-none-any.whl"
+                "input_file": "wheels/shared/slack_sdk-3.33.5-py2.py3-none-any.whl"
```

Comment:
Have you tried removing the whole `pip_dependencies` key?

Reply:
I have removed the pip_dependencies key, and tested on a fresh 6.3.0.693 instance. It's passing.